### PR TITLE
readme: describe pipeline variants

### PR DESCRIPTION
**Pull request by sdurrheimer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,42 @@ In order to run this task, couple elements are required within the infrastructur
   * `terraform-apply`: Terraform job similar to the plan one, but will actually create/update everything that needs to. Please see the plan diff for a better understanding.
   * `terraform-destroy`: :warning: Terraform job meant to destroy the whole stack - **NO CONFIRMATION ASKED**. If triggered, the full project **WILL** be destroyed. Use with caution.
 
+**Variations**
+
+There is 4 versions of the pipeline supported in this stack:
```
Comment (tormath1) [inline README.md]:
```suggestion
There are 4 versions of the pipeline supported in this stack:
```

File: README.md [hunk 2]
```diff
@@ -26,8 +26,42 @@ In order to run this task, couple elements are required within the infrastructur
...
+  * AWS: `pipeline.yml` + `variables.sample.yml` (default)
+  * S3-like compatible backends: `pipeline-s3-compatible.yml` + `variables-s3-compatible.sample.yml`
+  * GCP: `pipeline-gcp.yml` + `variables-gcp.sample.yml`
+  * Azure: `pipeline-azure.yml` + `variables-azure.sample.yml`
+
+Currently, Cycloid UI doesn't support switching between different pipeline variations, you will have to manually copy+paste pipeline+variables from GIT inside the Cycloid UI to use them.
```
Comment (tormath1) [inline README.md]: Maybe we should talking about `Cycloid console` instead of `Cycloid UI` ?

**Comment by tormath1** [comment]
LGTM !

File: README.md (hunk 2 above)
Comment (sdurrheimer) [reply README.md]: yeah sure

**Review by tormath1 — APPROVED** [review]

**Pull request merged**